realsense2_camera/test/utils/pytest_live_camera_utils.py

```diff
@@ -100,12 +107,15 @@ def get_color_profiles(long_data, start_index, end_index):
         if len(color_profile) == 5:
             profile = color_profile[0]
             value = color_profile[1]+"x"+color_profile[3]
-            format = color_profile[4]
+            img_format = color_profile[4]
+        elif len(color_profile) == 7:
```

Comment:
If just `rs-enumerate` output format was changed, why do we need this extra `if's`?

Reply:
Oh, I understand..
`color_profile` is not acctually color profile, it's a splitted line from rs-enumerate?
Maybe we can rename it and maybe add const vars for the indexes with the actual meaning?
example

```static const size_t color_res_width = 1;
static const size_t color_res_width = 3;```

Reply:
Good point ! 
Thank you, We shall change.